tests_integration_with_local_chain/conftest.py

```diff
@@ -41,12 +46,14 @@ def local_web3(
         fund_account_on_tenderly(
             tenderly_fork_rpc, [a.address for a in accounts], xDai(1000)
         )
+        set_env_rpc_url(tenderly_fork_rpc)
```

Comment:
Why is this needed now? Can't we continue passing `web3` instances around?

Reply:
If `RPCConfig` is initialised somewhere in the code, it didn't pick up the localchain rpc URL, but the production one from ENV.

<img width="561" alt="image" src="https://github.com/user-attachments/assets/88145108-eca4-41ff-80e5-c96681e98a79" />